klaviyo/php-sdk/2.0/config/packages/klaviyo.yaml

```diff
@@ -0,0 +1,6 @@
+services:
+    Klaviyo\Klaviyo:
+        class: Klaviyo\Klaviyo
```

Comment:
This row is not needed when you have the service ID as: "Klaviyo\Klaviyo"

Reply:
I didn't know, thanks for the tip.